tango_ros_common/tango_ros_native/src/occupancy_grid_file_io.cpp

```diff
@@ -27,9 +29,47 @@
 namespace {
 void AddTrailingSlashToDirectoryPathIfNeeded(std::string& directory_path) {
   if (!directory_path.empty() && directory_path.back() != '/') {
-    directory_path += "/";
+    directory_path += '/';
   }
 }
+bool DirectoryPathExists(const std::string& path) {
```

Comment:
These functions are very generic so deserve their own file: file_io.cpp. This is only an opinion so ok to move them later, if they are reused elsewhere in the code.